packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/drawable/CSSBackgroundDrawable.java

```diff
@@ -1105,7 +1120,11 @@ private void drawRectangularBackgroundWithBorders(Canvas canvas) {
     mPaint.setStyle(Paint.Style.FILL);
 
     int useColor = multiplyColorAlpha(mColor, mAlpha);
-    if (Color.alpha(useColor) != 0) { // color is not transparent
+    if (mGradients != null && mGradients.length > 0) {
+      mPaint.setShader(getGradientShader());
+      canvas.drawRect(getBounds(), mPaint);
+      mPaint.setShader(null);
+    } else if (Color.alpha(useColor) != 0) {
```

Comment:
Not important for this change, but I think we want to support background color and images at the same time.

https://www.w3.org/TR/css-backgrounds-3/#background-color

> This property sets the background color of a box. This color is drawn behind any background images.

Reply:
> This color is drawn behind any background images.

True, letting it get drawn behind image should be good.

e.g. also tested, below works fine.
```
if (Color.alpha(useColor) != 0) {}
if (mGradients != null && mGradients.length > 0) {}
```

Reply:
Updated. https://github.com/facebook/react-native/pull/45433/commits/2f1bf9eef3a285ee7e43208170ab06e9a5851d2b.